### DISCUSSION TITLE
[Paper Review 13][25.12]ROOT: Robust Orthogonalized Optimizer for Neural Network Training

### DISCUSSION THREAD

**Discussion by EuijuHeo** [Pseudo-Lab/SOTA-AI-Review#15] [General]
논문 제목: ROOT: Robust Orthogonalized Optimizer for Neural Network Training
논문 링크: https://huggingface.co/papers/2511.20626 (upvote 162, 25.11.30 기준)

논문 요약:
Motivation
- Optimizer의 선택은 수렴 속도와 최종 성능 뿐만 아니라 모델 개발의 막대한 경제적 비용에 깊은 영향을 미침
- 기존 Optimizer의 수치적 불안정성, 차원적 취약성, 이상치 민감도를 해결하는 새로운 Optimizer 제안

Core Methodology
1) 알고리즘적 Robustness: ROOT는 Muon의 고정 계수 뉴턴-슐츠 반복을 차원 별 계수를 사용하는 적응형 뉴턴 반복으로 대체
→ 다양한 아키텍처 구성에서 일관된 정밀도 보장, 행렬 차원 변화에 강건하게 만듦
2) 최적화 Robustness: 데이터 수준 노이즈에 대한 최적화 강건성을 위한 근접 최적화(proximal optimization) 프레임워크 도입
→ Soft-Thresholding을 통해 이상치 유발 Gradient noise를 억제, 훈련을 안정화

Experimental Results
- Muon 및 Adam 기반 Optimizer 대비 더 빠르고 안정적인 수렴을 달성
- 특히 노이즈가 많고 비대칭형 차원에서도 우수한 최종 성능 입증
- ROOT의 AdaNewton은 Muon의 고정 계수 방식보다 직교화 정밀도를 크게 향상
- ROOT의 Soft-threshold 기반 Outlier 억제 알고리즘은 LLM 및 비전 작업 모두에서 일반화 성능을 향상시킴
